src/CheckoutSdk/Sessions/Channel/ChannelData.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json;
+
+namespace Checkout.Sessions.Channel
+{
+    public abstract class ChannelData
+    {
+        [JsonProperty(PropertyName = "channel")]
```

Comment:
Maybe "[JsonProperty(PropertyName = "channel")]" it isn't necessary

Reply:
Is kind of necessary because his name starts with underscore.